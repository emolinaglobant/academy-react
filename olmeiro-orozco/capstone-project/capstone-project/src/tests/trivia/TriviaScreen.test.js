const { shallow } = require("enzyme");
const { TriviaScreen } = require("../../components/trivia/TriviaScreen");
import '@testing-library/jest-dom';

describe('Tests component <TriviaScreen />', () => {

  test('should render component correctly', () => { 
    const wrapper = shallow(<TriviaScreen />);

    expect(wrapper).toMatchSnapshot();
   })
})