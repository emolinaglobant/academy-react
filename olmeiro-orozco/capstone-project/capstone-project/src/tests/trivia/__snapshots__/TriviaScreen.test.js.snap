// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests component <TriviaScreen /> should render component correctly 1`] = `
<section
  className="trivia__main-content"
>
  <HeaderGame />
  <section
    className="trivia__main-central"
  >
    <SidebarLeft />
    <CentralGame />
    <SidebarRight />
  </section>
  <FooterGame />
</section>
`;
